diff --git a/cuda_rasterizer/rasterizer_impl.h b/cuda_rasterizer/rasterizer_impl.h
index 638bd4c..ac4e7d4 100644
--- a/cuda_rasterizer/rasterizer_impl.h
+++ b/cuda_rasterizer/rasterizer_impl.h
@@ -13,6 +13,7 @@
 
 #include <iostream>
 #include <vector>
+#include <cstdint>
 #include "rasterizer.h"
 #include <cuda_runtime_api.h>
 
@@ -74,4 +75,4 @@ namespace CudaRasterizer
 		T::fromChunk(size, P);
 		return ((size_t)size) + 128;
 	}
-};
\ No newline at end of file
+};
diff --git a/setup.py b/setup.py
index 8892029..0fdb767 100644
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,7 @@ setup(
             "cuda_rasterizer/backward.cu",
             "rasterize_points.cu",
             "ext.cpp"],
-            extra_compile_args={"nvcc": ["-I" + os.path.join(os.path.dirname(os.path.abspath(__file__)), "third_party/glm/")]})
+            extra_compile_args={"nvcc": ["-I" + os.path.join(os.path.dirname(os.path.abspath(__file__)), "third_party/glm/"), "-DGLOG_USE_GLOG_EXPORT"], "cxx": ["-DGLOG_USE_GLOG_EXPORT"]})
         ],
     cmdclass={
         'build_ext': BuildExtension
